packages/devui-vue/devui/popover/__tests__/popover.spec.tsx

```diff
@@ -56,11 +63,9 @@ describe('d-popover', () => {
       },
     });
     await wrapper.find(buttonBaseClass).trigger('mouseenter');
-    setTimeout(() => {
-      const popoverContent = document.body.querySelector(popoverContentClass);
-      expect(popoverContent).toBeTruthy();
-      wrapper.unmount();
-    }, 150);
+    await new Promise(resolve => setTimeout(resolve, 500));
```

Comment:
@hxj9102 
```suggestion
    await wait(500);
```

Reply:
good suggestion！already changed and commited.